slides/developer-guide/working-with-files-folders-and-storage/working-with-folders/_index.md

```diff
@@ -760,10 +760,10 @@ class Application
         SlidesApi slidesApi = new SlidesApi("MyClientId", "MyClientSecret");
 
         string storageName = "MyStorage";
-        string folderPath = "MyFolder/MyPresentations";
+        string folderToDelete = "MyFolder/MyPresentations";
```

Comment:
This is a very simple code example and in this case, such clarifications are unnecessary.